sdk/monitor/opentelemetry-exporter-azuremonitor/opentelemetry/exporter/azuremonitor/protocol.py

```diff
@@ -0,0 +1,678 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
```

Comment:
Do we still need this file now that we are using `models`?

Reply:
Removed